python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -964,7 +964,7 @@ def aten〇embedding_bag〇padding_idx(weight: List[int], indices: List[int], of
     output_bag_shape.append(out_dim1)
 
     offset2bag_shape: List[int] = []
-    if mode == 1:
+    if mode == 0:
```

Comment:
nit: can you create variables with the names of the modes, to make this easier to read? i.e.

```python
sum_mode = 0
...

if mode == sum_mode:
  ...
```

Reply:
yes